vars/rollingUpgradePipeline.groovy

```diff
@@ -111,19 +139,6 @@ def call(Map pipelineParams) {
                                                 }
                                             }
                                         }
-                                        stage("Get test duration for ${base_version}") {
```

Comment:
move this out of here, assume that every test in the parallel are with same exact configuration...
I might not be the case. I wouldn't move this stage...

Reply:
As I can see, there is no dependency on base version for this stage. Why do we need to change global variables in every parallel stage?

Look here:

https://github.com/scylladb/scylla-cluster-tests/blob/bc931a0a7769d2c52f9610d727f13134f087cad1/vars/getJobTimeouts.groovy#L6-L11

Reply:
I've forgot those are assigned to globals...  never mind then..